hack/gofmt.sh

```diff
@@ -12,7 +12,7 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   cd /tmp/unit
   make fmt > /tmp/fmt-output.log
   FILE_LENGTH="$(wc -l /tmp/fmt-output.log | awk '{ print $1 }')"
-  if [ "${FILE_LENGTH}" != "1" ]; then
+  if [ "${FILE_LENGTH}" != "2" ]; then
```

Comment:
why did this change ?

Reply:
The gofmt is failing otherwise since we are doing fmt twice once in root folder once in /api

Reply:
@maelk since we are doing go fmt inside the /api folder in Makefile, without above change gofmt fails with:
```
+ wc -l /tmp/fmt-output.log
+ awk { print $1 }
+ FILE_LENGTH=2
+ [ 2 != 1 ]
+ echo Formatting error! Please run 'make fmt' to correct the problem.
Formatting error! Please run 'make fmt' to correct the problem.
+ echo The problematic files are listed below, after the command that should be run
The problematic files are listed below, after the command that should be run
+ cat /tmp/fmt-output.log
go fmt ./controllers/... ./baremetal/... .
cd api; go fmt  ./...
+ exit 1
```
We thought that since we have run gofmt twice and FILE_LENGTH=2 as a result of that, the check for it was changed from 1 to 2. Does that make sense?

Reply:
running `make fmt` was passing fine and was not changing anything actually.

Reply:
yes, makes sense, thanks. Can you please add a short comment saying what those two lines are ?

Reply:
basically those two lines are the headers from two gofmt runs

Reply:
cc @kashifest

Reply:
@maelk resolved